S/SoXResampler/build_tarballs.jl

```diff
@@ -33,7 +32,7 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
+    Dependency(PackageSpec(name="OpenMPI_jll", uuid="fe0851c0-eecd-5654-98d4-656369965a5c"))
```

Comment:
Why did you remove `CompilerSupportLibraries_jll` if you want OpenMP?  Also, how is OpenMPI related?

Reply:
Well, removing compiler support libraries was an accident...I think libgomp and OpenMPI are different implementations of OpenMP? I added both to hedge my bet. I'm still not sure how to figure out whether something is a run-time or a compile-time dependency. The build reports:

```
[17:41:46] -- Found OpenMP_C: -fopenmp (found version "3.1") 
[17:41:46] -- Found OpenMP: TRUE (found version "3.1")
```

So one of them must have worked?

Reply:
> I think libgomp and OpenMPI are different implementations of OpenMP?

No, libgomp is an implementation of [OpenMP](https://en.wikipedia.org/wiki/OpenMP), OpenMPI is an implementation of [MPI](https://en.wikipedia.org/wiki/Message_Passing_Interface)

Reply:
> So one of them must have worked?

Target libgomp is _always_ there during the build, `CompilerSupportLibraries_jll` is more important for runtime

Reply:
It's already there? I tried Pkg.develop ing and I'm getting new errors in my in-progress PulseAudio build:

```
/opt/x86_64-linux-gnu/bin/../lib/gcc/x86_64-linux-gnu/4.8.5/../../../../x86_64-linux-gnu/bin/ld: warning: libgomp.so.1, needed by /workspace/destdir/lib/libsoxr.so.0, not found (try using -rpath or -rpath-link)
/workspace/destdir/lib/libsoxr.so.0: undefined reference to `omp_get_num_threads@OMP_1.0'
/workspace/destdir/lib/libsoxr.so.0: undefined reference to `GOMP_parallel_start@GOMP_1.0'
/workspace/destdir/lib/libsoxr.so.0: undefined reference to `omp_set_lock@OMP_3.0'
/workspace/destdir/lib/libsoxr.so.0: undefined reference to `omp_destroy_lock@OMP_3.0'
/workspace/destdir/lib/libsoxr.so.0: undefined reference to `omp_get_thread_num@OMP_1.0'
/workspace/destdir/lib/libsoxr.so.0: undefined reference to `omp_init_lock@OMP_3.0'
/workspace/destdir/lib/libsoxr.so.0: undefined reference to `GOMP_parallel_end@GOMP_1.0'
/workspace/destdir/lib/libsoxr.so.0: undefined reference to `omp_unset_lock@OMP_3.0'
```

Reply:
> `try using -rpath or -rpath-link`

Reply:
It's really easier for me if we're on the same page and you push whatever changes you have locally.

Reply:
Sure! Sorry I'm so hopeless at this. Current try is at #2469